### PR TITLE
Add a definition of what is a module compatible multistore

### PR DESCRIPTION
**Pull request by marionf**
Add a definition of what is a module compatible multistore

### CODE REVIEW

File: src/content/1.7/modules/creation/_index.md [hunk 1]
```diff
@@ -447,6 +447,11 @@ for their own settings.
 
 ### Handling the Multistore feature
 
+What is a module compatible multistore ?
```
Comment (PierreRambaud) [inline src/content/1.7/modules/creation/_index.md]:
```suggestion
What is a module compatible with the multi-store feature?
```
You can't use one time `multistore` and next time `multi-store`

File: src/content/1.7/modules/creation/_index.md [hunk 2]
```diff
@@ -447,6 +447,11 @@ for their own settings.
...
+A multi-store compatible module is a module that can be configured differently from one store to another.
```
Comment (PierreRambaud) [inline src/content/1.7/modules/creation/_index.md]:
```suggestion
It is a module that can be configured differently from one store to another.
```

File: src/content/1.7/modules/creation/_index.md [hunk 3]
```diff
@@ -447,6 +447,11 @@ for their own settings.
...
+It is also a module that can be configured quickly in the same way on all stores thanks to the all shops context or the group of shops.
```
Comment (PierreRambaud) [inline src/content/1.7/modules/creation/_index.md]:
```suggestion
Also the same module can have a shared configuration between all shops.
```
wdyt?

**Comment by matks** [comment]
Thank you @marionf

**Pull request merged**